src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplAPIDocumentationSteps.java

```diff
@@ -0,0 +1,68 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import static org.testng.Assert.assertTrue;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import gov.healthit.chpl.aqa.pageObjects.BasePage;
+import gov.healthit.chpl.aqa.pageObjects.ChplAPIPage;
+
+/**
+ * Class ChplAPIDocumentationSteps definition.
+ */
+public class ChplAPIDocumentationSteps extends BaseSteps {
+
+    /**
+     * Constructor creates new driver.
+     */
+    public ChplAPIDocumentationSteps() {
+        super();
+       }
+
+    /**
+     * Get user to the API Documentation page.
+     */
+    @Given("^I am on CHPL API page$")
+    public void userLoadsAPIPage() {
+        getDriver().get(getUrl() + "#/resources/chpl_api");
+        WebDriverWait wait = new WebDriverWait(getDriver(), TIMEOUT);
+        wait.until(ExpectedConditions.visibilityOf(BasePage.mainContent(getDriver())));
```

Comment:
```suggestion
        wait.until(ExpectedConditions.visibilityOf(ChplAPIPage.mainContent(getDriver())));
```
With ChplApiPage extending BasePage, you can treat the ChplApiPage as having all of the BasePage content